CommunityProject/Assets/_Game/Scripts/Editor/OnBuildDestroyObjsManager.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace BoundfoxStudios.CommunityProject.Editor
+{
+	public class OnBuildDestroyObjsManager : IProcessSceneWithReport
```

Comment:
```suggestion
	public class StripDevelopmentOnlyObjectsFromBuild : IProcessSceneWithReport
```

Bitte den Dateinamen auch mit anpassen :)